extensions-core/datasketches/src/main/java/org/apache/druid/query/aggregation/datasketches/hll/HllSketchMergeAggregator.java

```diff
@@ -0,0 +1,94 @@
+/*
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.apache.druid.query.aggregation.datasketches.hll;
+
+import com.yahoo.sketches.hll.Union;
+import com.yahoo.sketches.hll.HllSketch;
+import com.yahoo.sketches.hll.TgtHllType;
+
+import org.apache.druid.query.aggregation.Aggregator;
+import org.apache.druid.segment.ColumnValueSelector;
+
+/**
+ * This aggregator merges existing sketches.
+ * The input column must contain {@link HllSketch}
+ * @author Alexander Saydakov
+ */
+public class HllSketchMergeAggregator implements Aggregator
+{
+
+  private final ColumnValueSelector<HllSketch> selector;
+  private final TgtHllType tgtHllType;
+  private Union union;
+
+  public HllSketchMergeAggregator(final ColumnValueSelector<HllSketch> selector, final int lgK, final TgtHllType tgtHllType)
```

Comment:
Line longer than 120 cols. Please fix other violations in the PR

Reply:
fixing